### ISSUE TITLE
Simple math using variables in Waypoint Calculator

### ISSUE THREAD

**Issue by CharlieTemplar** [cgeo/cgeo#8105]
**Is your feature request related to a problem?**
Waypoint Calculator returns '*' suggesting invalid input when doing simple math with variables.
![image](https://user-images.githubusercontent.com/47461175/73137530-19f16300-4051-11ea-830f-38f4d66dc5ce.png)

**Describe the solution you'd like**
I would like to be able to do simple math using these main variables.  I know that there are sub variables which can be used for math, but the also return * when referring to other variables.
I'd be happy if there is a way to do this already, especially without too much complication.

**Describe alternatives you've considered**
use sub variables, but this also comes back as invalid

**Additional context**
See screenshot

**Comment by Lineflyer** [comment]
@eddiemuc Here is another issue regarding the legacy waypoint calculator which IMHO becomes obsolete if we get rid of these "sub variables". Correct?
Then we should close here, as its implicitly solved once the new calc is ready.

**Comment by eddiemuc** [comment]
Correct

**Comment by CharlieTemplar** [comment]
Ignored for 2 years now closed as obsolete...  Where is the replacement/fix? Which version does this come in and how does it solve my feature request?

**Comment by SammysHP** [comment]
There will be a completely new calculator in one of the next releases that will support what you requested.

**Comment by eddiemuc** [comment]
@CharlieTemplar the new calculator is in early development state and thus yet only available in nightlies. But it is already in master code branch, which is the point where we usually close related issues.

This is a really complex feature and I will need some more time to finish it.

Would you prefer to leave this open for your personal tracking?

**Comment by CharlieTemplar** [comment]
I'll be honest I gave up and forgot about this.  It's good to get some indication that this being addressed.  
I just tried the latest Nightly and this looks/works great except the W/E is ignored and always seems go East but as you said it's not finished yet.

![Screenshot_20211226-160419_cgeo](https://user-images.githubusercontent.com/47461175/147414016-90544311-9c89-4ade-b5ea-500a5ce4e148.jpg)
